.github/i18n/README.es.md

```diff
@@ -0,0 +1,423 @@
+[![Docker](https://github.com/jina-ai/jina/blob/master/.github/badges/docker-badge.svg?raw=true  "Jina is multi-arch ready, can run on different architectures")](https://hub.docker.com/r/jinaai/jina/tags)
+[![Docker Image Version (latest semver)](https://img.shields.io/docker/v/jinaai/jina?color=%23099cec&label=Docker%20Image&logo=docker&logoColor=white&sort=semver)](https://hub.docker.com/r/jinaai/jina/tags)
+[![CI](https://github.com/jina-ai/jina/workflows/CI/badge.svg)](https://github.com/jina-ai/jina/actions?query=workflow%3ACI)
+[![CD](https://github.com/jina-ai/jina/workflows/CD/badge.svg?branch=master)](https://github.com/jina-ai/jina/actions?query=workflow%3ACD)
+[![Release Cycle](https://github.com/jina-ai/jina/workflows/Release%20Cycle/badge.svg)](https://github.com/jina-ai/jina/actions?query=workflow%3A%22Release+Cycle%22)
+[![Release CD](https://github.com/jina-ai/jina/workflows/Release%20CD/badge.svg)](https://github.com/jina-ai/jina/actions?query=workflow%3A%22Release+CD%22)
+[![API Schema](https://github.com/jina-ai/jina/workflows/API%20Schema/badge.svg)](https://api.jina.ai/)
+[![codecov](https://codecov.io/gh/jina-ai/jina/branch/master/graph/badge.svg)](https://codecov.io/gh/jina-ai/jina)
+
+</p>
+
+<p align="center">
+  <a href="https://github.com/jina-ai/jina">English</a> •
+  <a href="https://github.com/jina-ai/jina/blob/master/README.ja.md">日本語</a> •
+  <a href="https://github.com/jina-ai/jina/blob/master/README.fr.md">Français</a> •
+  <a href="https://github.com/jina-ai/jina/blob/master/README.de.md">Deutsch</a> •
+  <a href="https://github.com/jina-ai/jina/blob/master/README.ru.md">Русский язык</a> •
+  <a href="https://github.com/jina-ai/jina/blob/master/README.zh.md">中文</a> •
+  <a href="https://github.com/jina-ai/jina/blob/master/README.pt_br.md">Português (BR)</a>
+  <a href="https://github.com/jina-ai/jina/blob/master/README.es.md">Español</a>
+</p>
+
+
+<p align="center">
+  <a href="https://jina.ai">Sitio Web</a> •
+  <a href="https://docs.jina.ai">Documentos</a> •
+  <a href="https://learn.jina.ai">Ejemplos</a> •
+  <a href="https://github.com/jina-ai/jina-hub">Hub (beta)</a> •
+  <a href="https://dashboard.jina.ai">Dashboard (beta)</a> •
+  <a href="https://github.com/jina-ai/jinabox.js/">Jinabox (beta)</a> •
+  <a href="http://www.twitter.com/JinaAI_">Twitter</a> •
+  <a href="https://jobs.jina.ai">Estamos contratando</a>
+</p>
+
+Jina es un framework de búsqueda basado en la IA que permite a los desarrolladores crear sistemas de búsqueda **cross/multi-modals** (como texto, imágenes, video, audio) en la nube.
+
+⏱️ **Ahorro de tiempo** - Inicie un sistema alimentado por la IA en sólo unos minutos..
+
+🧠 **Modelos de IA de primeira classe** - *El* patrón de diseño de los sistemas de búsqueda neuronal, con soporte de primera clase para [modelos de IA de última generación](https://docs.jina.ai/chapters/all_exec.html).
```

Comment:
Hey Pablo, there are a couple of typos here

Reply:
Hi Joan thank you for your support. Please what is the best way to fix it? Committing the changes to the same branch os ok? What about the commit title? Glad to contribute.